index.js

```diff
@@ -16,7 +16,12 @@ function sum(arr) {
   return arr[0] + sum(arr.slice(1));
 }
 
+function fibonacci(num){
+  let arr=[]
+  for(let i=0; i<=num; i++)
```

Comment:
Non hai bisogno di utilizzare un `for` in questo caso. 
Basta utililizzare la ricorsione senza l'utilizzo di alcun ciclo.